test/utility/backup.c

```diff
@@ -274,6 +274,48 @@ __int_comparator(const void *a, const void *b)
     return (*(int *)a - *(int *)b);
 }
 
+/*
+ * testutil_last_backup_id --
+ *     Find the last valid backup id number. Can be used after a restart to determine backups based
+ *     on directory naming.
+ */
+void
+testutil_last_backup_id(int *last_full, int *last_incr)
```

Comment:
`last_incr` is not used by the caller.  Is that something that will be supported soon?  If not, we shouldn't introduce dead code.

Reply:
Yes, good call. If it is ever needed, then it can be added.